openshift_scalability/README.md

```diff
@@ -175,6 +175,64 @@ some combintations of access modes will not work. Eg. having for PV `ReadWriteMa
 
 Example of config file with storage extension is `storagepyconf.yaml` 
 
+## Using cluster-loader with --auto-gen for dynamic JMeter pod creation
+
+Cluster Loader has the functionality to dynaimcally create load generating pods that target pre-existing applications
+on the OSE cluster. There are only a few minor changes to make to the config so the correct parameters are present,
+as well as the command line flag.
+
+### Sample Command
+```
+ $ python cluster-loader.py -af ./config/stress.yaml
+
+```
+
+To utilize the environment detection function you must pass the -a/--auto-gen flag (as above).
+
+
+### Sample Config File
+```
+projects:
+  - num: 1
+    basename: centos-stress
+    tuning: default
+    templates:
+      - num: 1
+        file: ./content/quickstarts/stress/template-pod.json
```

Comment:
Perhaps you meant ./content/quickstarts/stress/stress-pod.json? Or really just a generic "./content/quickstarts/stress/template-pod.json" which doesn't exist, but stresses it is a template?